xca.json

```
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://www.hohnstaedt.de/xca/",
+    "description": "X - Certificate and Key management",
+    "license": "BSD-3-Clause",
+    "version": "2.1.2",
+    "url": "https://github.com/chris2511/xca/releases/download/RELEASE.2.1.2/xca-portable-2.1.2.zip",
+    "hash": "37d2e315dc36c7060a50dd7203339dfa8c8b43cea34f340d456a900a6e5232a6",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse",
```

Remove redundant post_install after change to zip

```suggestion
    "extract_dir": "xca-portable-2.1.2",
```